corehq/apps/userreports/sql/adapter.py

```diff
@@ -149,8 +149,6 @@ def save_rows(self, rows, use_shard_col=True):
         Saves rows to a data source after deleting the old rows
         :param use_shard_col: use shard column along with doc id for searching rows to delete/update
         """
-        if not self.table_exists:
```

Comment:
Turns out we expect it to fail if this condition is not met, so removing this piece.